Directory.Build.targets

```diff
@@ -0,0 +1,60 @@
+<Project>
+  <ItemGroup Condition="'$(IncludeGlobalAssemblyInfo)' == 'true'">
+    <PackageReference Include="GitInfo" Version="2.0.20" PrivateAssets="All" />
+		<Compile Include="..\Xamarin.Forms.Core\Properties\GlobalAssemblyInfo.cs" Link="Properties\GlobalAssemblyInfo.cs" />
+	</ItemGroup>
+
+  	<!-- Local builds should always be commit count +1, since we'll be always working on the next version to be published. -->
+	<Target Name="_SetLocalCommits" BeforeTargets="_GitPopulateVersionInfo" Condition="'$(CI)' != 'true'">
+		<PropertyGroup>
+			<GitCommits>$([MSBuild]::Add('$(GitCommits)', '1'))</GitCommits>
+		</PropertyGroup>
+	</Target>
+
+  <Target Name="SetGitBranch" AfterTargets="GitInfo" Condition="'$(CI)' == 'true'">
```

Comment:
You don't really need to condition this and run in in a target. Those envvars will only ever exist on CI anyway. Example: https://github.com/kzu/AutoCodeFix/blob/master/src/Version.targets#L14-L18